src/stores/utils/web3.js

```diff
@@ -79,12 +95,16 @@ export const estimateGas = async (web3, to, gasPrice, from, value, data) =>{
 const processWeb3 = (web3, resolve,  reject) => {
   web3.version.getNetwork((err, netId) => {
     const netIdName = getNetworkName(netId)
-    console.log(`This is ${netIdName} network.`, netId)
-    document.title = `${netIdName} - Bridge UI dApp`
```

Comment:
Please check the case what will be displayed when MM/NW is not installed or it is still locked.

Reply:
If I disable _MM/NW_, or the dApp is still locked I'm getting 'dApp .env Title - No chain specified' (that's according to my local .env file, of course).

![Screen Shot 2019-03-27 at 16 52 30](https://user-images.githubusercontent.com/4015436/55107753-d59fda00-50b0-11e9-99e0-87003b55e6dd.png)

Let me know if there's another case.

Reply:
Is this after https://github.com/poanetwork/bridge-ui/pull/201/commits/ca78124b39f4a268a38c66fdab5119e8fa81be54 applied?

Reply:
At this point this branch is up to date with the latest changes on /develop (resolved conflicts and all).